apps/web/pages/api/username/index.ts

```diff
@@ -0,0 +1,15 @@
+import type { NextApiResponse } from "next";
+
+// @TODO: move to packages/lib so it can be used on web/website
+import usernameHandler, { RequestWithUsernameStatus } from "../../../../website/lib/username";
```

Comment:
We cannot leave this like this. This will cause errors on self-hosted users since they don't have this module. Let's just stick to calling the `cal.com/api/username` endpoint directly from here.

Reply:
Yeah this change was before you helped me the other day with the same issue. Would keep this as a reminder to use api endpoint